Workbooks/AKS/Subnet IP Usage/Subnet IP Usage.workbook

```diff
@@ -483,18 +491,21 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "InsightsMetrics\n| where TimeGenerated < {timeRange:end} and TimeGenerated >= {timeRange:start}\n| where Name == \"cx_ipam_pod_allocated_ips\"\n| where Namespace == \"prometheus\"\n| extend jsonprop = parse_json(Tags)\n| extend HostName = jsonprop.hostName\n{hostwhereclause}\n| extend podnet_arm_id = jsonprop.podnet_arm_id\n| project TimeGenerated, Val, podnet_arm_id, HostName\n| summarize count = count(Val) by bin(TimeGenerated, {timeRange:grain}), tostring(HostName)\n\n",
+              "query": "InsightsMetrics\n| where TimeGenerated < {timeRange:end} and TimeGenerated >= {timeRange:start}\n| where Name == \"cx_ipam_pod_allocated_ips\"\n| where Namespace == \"prometheus\"\n| extend jsonprop = parse_json(Tags)\n| extend HostName = jsonprop.hostName\n{hostwhereclause}\n| project TimeGenerated, Val, HostName\n| summarize count = count(Val) by bin(TimeGenerated, {timeRange:grain}), tostring(HostName), Val\n\n",
```

Comment:
why are we summarizing by 'Val' ?

Reply:
We were summarizing by count(Val). The metric values were off. Removed that and using only Val.